src/PrestaShopBundle/Form/Admin/Sell/Product/Specification/SpecificationsType.php

```diff
@@ -99,7 +99,7 @@ public function configureOptions(OptionsResolver $resolver)
         $resolver
             ->setDefaults([
                 'required' => false,
-                'label' => false,
+                'label' => $this->trans('Specifications', 'Admin.Catalog.Feature'),
```

Comment:
```suggestion
                'label' => $this->trans('Details', 'Admin.Catalog.Feature'),
```

We decided to name that tab "Details", see mockups: 
![image](https://user-images.githubusercontent.com/70583503/182819441-0fb6b277-f03e-4ef7-aa69-8b6465a334e3.png)

Reply:
Yeah I know I planned to change this in a dedicated PR because it implies changing the form structure as well, but since you started the catalog export I will modify this wording right now and handle the form structure independently later